network/src/proto/mempool.proto

```diff
@@ -5,14 +5,54 @@ syntax = "proto3";
 
 package mempool;
 
+import "mempool_status.proto";
 import "transaction.proto";
+import "vm_errors.proto";
 
 /* MempoolSyncMsg represents the messages exchanging between validators to keep
  * transactions in sync. The proto definition provides the spec on the wire so
  * that others can implement their mempool service in various languages.
  * Mempool service is responsible for sending and receiving MempoolSyncMsg
  * across validators. */
 message MempoolSyncMsg {
+  oneof message {
+    SubmitTransactionsRequest submit_transactions_request = 1;
```

Comment:
nit: what do you think about renaming it into BroadcastTransactionsRequest instead. Cause in case of Validator to Validator communication, it doesn't feel like submit is precise definition